src/components/FilterableProductTable.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import ProductTable from './ProductTable';
+import SearchBar from './SearchBar';
+
+export default class FilterableProductTable extends Component {
```

Comment:
Great, great job mate! Just missing the checkbox that I am sure you can do it. 🥇